docs/gaps/gaps.tex

```diff
@@ -0,0 +1,286 @@
+\documentclass{article}
+
+%%%% packages and definitions (optional)
+\usepackage{graphicx} % allows inclusion of graphics
+\usepackage{graphics}
+\usepackage{placeins}
+\usepackage{booktabs} % nice rules (thick lines) for tables
+\usepackage{microtype} % improves typography for PDF
+\usepackage{xspace}
+\usepackage[hidelinks]{hyperref}
+\usepackage{xspace}
+\usepackage{hhline}
+
+\usepackage{tabularx}
+\newcolumntype{b}{X}
+\newcolumntype{s}{>{\hsize=.5\hsize}X}
+\newcolumntype{m}{>{\hsize=.75\hsize}X}
+
+\newcommand{\SN}{S$_N$}
+\renewcommand{\vec}[1]{\bm{#1}} %vector is bold italic
+\newcommand{\vd}{\bm{\cdot}} % slightly bold vector dot
+\newcommand{\grad}{\vec{\nabla}} % gradient
+\newcommand{\ud}{\mathop{}\!\mathrm{d}} % upright derivative symbol
+\newcommand{\Cyclus}{\textsc{Cyclus}\xspace}%
+\newcommand{\Cycamore}{\textsc{Cycamore}\xspace}%
+\graphicspath{ {images/} }
+\usepackage[affil-it]{authblk}
+\usepackage{tikz}
+\usetikzlibrary{positioning, arrows, decorations, shapes }
+\usepackage{cleveref}
+
+
+\usepackage[acronym,toc]{glossaries}
+\include{acros.tex}
+	
+\makeglossaries
+
+\title{Non-algorithmic Capability Gaps for \Cyclus and \Cycamore for transition analyses }
+\author{Jin Whan Bae, Kathryn Huff}
+\affil{Dept. of Nuclear, Plasma, and Radiological Engineering, University of Illinois at Urbana-Champaign
+		  Urbana, IL}
+\date{}                     %% if you don't need date to appear
+\setcounter{Maxaffil}{0}
+\renewcommand\Affilfont{\itshape\small}
+%%%%%%%%%%%%%%actual words%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%5
+
+
+\begin{document}
+
+\section{Introduction}
+As part of NEUP-FY16-10512 (PI: Dr. Kathryn Huff), a test-run of fuel cycle
+transition scenarios was done using \Cyclus. The purpose of this study
+is to identify the non-algorithmic capability gaps necessary for a complete simulation.
+The fuel cycle models are from the Idaho National Laboratory Nuclear Fuel Cycle Evaluation and Screening Report
```

Comment:
I wouldn't call the evaluation groups "fuel cycle models". I would call them scenarios, evaluation scenarios, or evaluation groups. If you use evaluation groups, then I'd explain the term.